packages/grafana-data/src/types/plugin.ts

```diff
@@ -123,6 +123,7 @@ export interface PluginInclude {
 interface PluginMetaInfoLink {
   name: string;
   url: string;
+  href?: '_blank' | '_self' | '_parent' | '_top';
```

Comment:
Hmm, I think we should rather call this property `target`, however I am not sure I would expose all these options (or at least not until it's really necessary).

If we need to introduce a new property to the type I would rather go with something like `openInNewTab: boolean;`, which can default to `true` to reserve the current behaviour, or we can also update the `getEnterprisePhantomPlugins()` and set it explicitly everywhere.

Reply:
@leventebalogh agree on the href -> target part, no idea why I called it href 🤷 

But why not expose it / stick to the HTML standards - are we afraid of it being misused somehow?

Reply:
Changed to href but happy to change to `openInNewTab`, just want to understand it better.

https://github.com/grafana/grafana/pull/63499/commits/354a8c6f238f91977dc69c8e0de46fa202c19ca1

Reply:
Hmm, yea, I think I just wanted to be extra safe, but actually I can't imagine scenarios where `'_parent'` and `'_top'` wouldn't fall back to `'_self'`, so I think it's ok to allow it. 👍